docs/collections/_schema/human-readable-schema-grammar.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Human-Readable schema grammar
```

Comment:
Weird capitalization. I think you want "Human-readable schema grammar" here and elswhere

Reply:
Done.